packages/middleware-apply-body-checksum/src/applyMd5BodyChecksumMiddleware.ts

```diff
@@ -34,6 +34,10 @@ export function applyMd5BodyChecksumMiddleware(
           const hash = new options.md5();
           hash.update(body || "");
           digest = hash.digest();
+        } else if (isXmlNode(body)) {
```

Comment:
This case might not needed. As serializer output should already serialized to string or binaries